system-int-test/test/cmr/system_int_test/search/service/service_search_test.clj

```diff
@@ -2,16 +2,19 @@
   "This tests searching services."
   (:require
    [clojure.test :refer :all]
+   [cmr.common.mime-types :as mt]
    [cmr.common.util :refer [are3]]
    [cmr.mock-echo.client.echo-util :as e]
    [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.data2.umm-json :as du]
```

Comment:
Maybe change du to maybe data-umm-json?

Reply:
I am really not a fan of making the namespace alias, but apparently I am in the minority. I will update. :)